### PR TITLE
Fixed the issues of missing fields for CRD objects

### PR DESCRIPTION
**Pull request by jshaofuturewei**

### CODE REVIEW

**Review by CoderKevinZhang — APPROVED** [review]
LGTM

File: globalscheduler/test/yaml/sample_scheduler.yaml [hunk 1]
```diff
@@ -3,4 +3,42 @@ kind: Scheduler
 metadata:
   name: sample-scheduler
 spec:
-  tag: "1"
\ No newline at end of file
+  location:
```
Comment (kimeunju108) [inline globalscheduler/test/yaml/sample_scheduler.yaml:6]: This scheduler file is not matched with scheduler CRD

Comment (jshaofuturewei) [reply globalscheduler/test/yaml/sample_scheduler.yaml:6]: Please check it is a json format and match the following bold one.

type SchedulerSpec struct {
	// Location represent which geo location the scheduler is responsable for
	Location GeolocationInfo `json:"**location**"`

	// Tag represent the Nth scheduler object
	Tag string `json:"tag"`

	// Cluster is an array that stores the name of clusters
	Cluster []string `json:"cluster"`

	Union ClusterUnion `json:"union"`
}

**Review by kimeunju108 — APPROVED** [review]
samle_scheduler.yaml file is not matched with scheduler CRD
I approve this PR because this file is testing yaml file and not coding part.

**Comment by jshaofuturewei** [comment]
> samle_scheduler.yaml file is not matched with scheduler CRD
> I approve this PR because this file is testing yaml file and not coding part.

I double checked the scheduler json format after I applied the sample_scheduler.yaml to apiserver. The "location" works with no problem. The change is related to YAML not CRD.

"apiVersion": "v1",
    "items": [
        {
            "apiVersion": "globalscheduler.com/v1",
            ...
            "spec": {
                "cluster": [
                    "sample1",
                    "sample2"
                ],
                "location": {
                    "area": "Downtown",
                    "city": "Bellevue",
                    "country": "USA",
                    "province": "WA"
                },
                "tag": "1",

**Pull request merged**